src/components/JobLogsContainer/JobLogsContainer.js

```diff
@@ -35,24 +45,80 @@ const visibleColumns = [
 ];
 
 const JobLogsContainer = props => {
-  const { intl } = props;
+  const {
+    intl, stripes: { okapi },
+  } = props;
+
+  const calloutRef = useRef(null);
 
   const hasLoaded = true;
 
+  const handleDownloadError = () => {
+    if (!calloutRef.current) return;
+
+    calloutRef.current.sendCallout({
+      type: 'error',
+      message: <FormattedMessage id="ui-data-export.communicationProblem" />,
+    });
+  };
+
+  const getFileDownloadLink = async record => {
+    try {
+      const fileName = get(record.exportedFiles, '0.fileName');
+      const downloadLink = await getFileDownloadLinkRequest(record, okapi);
+
+      await downloadFile(fileName, downloadLink);
+    } catch (error) {
+      handleDownloadError();
+
+      console.error(error); // eslint-disable-line no-console
+    }
+  };
+
+  const downloadFile = (fileName, downloadLink) => {
+    const elem = window.document.createElement('a');
+
+    elem.href = downloadLink;
+    elem.download = fileName;
+    document.body.appendChild(elem);
+    elem.click();
+    document.body.removeChild(elem);
+  };
+
+  const getFileNameField = record => (
```

Comment:
What about the next case: the job has the Fail status. Should we have ability to download anything?

Reply:
Good question. To be honest, I don't know about this case. Creating the task, I hadn't such information about ability to download depends on the status.
@VictorSoroka1, could you clarify this case, please?

Reply:
@Igor-Gorchakov, @KVupp, please add your input. As far as I understand no file should be generated in such a case. For Q1 we only dealing with successful cases though.

Reply:
Correct, there will be no file present in S3 in case of an ERROR, and hence the API should not return any link

Reply:
Thanks!
I will make fileName of record with Fail status non-clickable.